src/core/Mixer.cpp

```diff
@@ -649,12 +657,12 @@ void Mixer::restoreAudioDevice()
 
 
 
-void Mixer::removeAudioPort( AudioPort * _port )
+void Mixer::removeAudioPort( AudioPort * port )
 {
 	requestChangeInModel();
 	QVector<AudioPort *>::Iterator it = std::find( m_audioPorts.begin(),
 							m_audioPorts.end(),
-							_port );
+							port );
```

Comment:
I'd like to see this entire statement get updated, removing newlines and spaces:
```
	QVector<AudioPort *>::Iterator it = std::find(m_audioPorts.begin(), m_audioPorts.end(), port);
```